Control_DataProcessingGUI.py

```diff
@@ -0,0 +1,101 @@
+from tkinter import *
+
+
+class DataDisplayControl:
+    def __init__(self):
+        pass
+
+    def set_graph1_array1(self, data):
```

Comment:
it seeeeeeems like to me that whatever is going to be implemented here could be done in a smarter way, like having a parameter for graph number and one for array number and only using one function. Unless the data going into each is going to be completely different.